README.md

```diff
@@ -22,6 +22,16 @@ secrets:
   ENVIRONMENT_VAR:
       secretRef: some-secret-reference
       key: connectionString
+keyVaults:
+  "cmc":
```

Comment:
is it possible to put a template in here for the environment?
that's what the pipeline is doing. (mostly because of some complaints about pipeline magic and it prevents accessing vaults that don't have the env in it or are subscription level vaults)